alertmanager-openapi/alertmanager-openapi.cabal

```diff
@@ -30,16 +30,16 @@ library
       lib
   ghc-options: -Wall -funbox-strict-fields
   build-depends:
-      aeson >=1.0 && <2.0
+      aeson >=2.0 && <3.0
```

Comment:
Annoying that this big version bump is necessary. I wish the OpenAPI people had supported multiple `aeson` versions, but it seems that the Aeson maintainers aren't supporting the `1.*` releases anymore.

Reply:
aeson 1.xx is susceptible to DoS attacks. It's possible to create a universal hash collision for FNV based hash algorithms.